docs/tutorials/Setting-up-Raspberry-Pi-Zero-W.md

```diff
@@ -0,0 +1,153 @@
+
+### Operating System
+Our tutorials assume that the operating system running on your Pi is *Raspbian Jessie* ([NOOBS](https://downloads.raspberrypi.org/NOOBS/images/NOOBS-2017-07-05/) or [image](https://downloads.raspberrypi.org/raspbian/images/raspbian-2017-07-05/)), not the more recent *Raspbian Stretch*.
+
+### Network
+Connect your Pi to the network, either over Wifi or with an Ethernet cable. We will use the network both to download required software to the Pi and to transfer compiled ELL models from your computer to the Pi. Raspberry Pi Zero has a Wifi module on board which makes it easy to connect over Wifi. If you are using a Raspberry Pi Zero with no integrated Wifi module, you may have to connect your board via a Ethernet cable or install a third part wifi dongle.
+
+### CMake
+We use CMake on the Raspberry Pi to create Python modules that can be called from our tutorial code. To install CMake on your Pi, connect to the network, open a terminal window, and type
+
+```shell
+sudo apt-get update
+sudo apt-get install -y cmake
+```
+
+### Python 3.4 installation
+Our tutorials require Python 3.4 on the Pi (and Python 3.6 on your computer). Since the last Conda support for ARMV6 is not actively maintained, here we install python 3.4 directly into our system path. We will also install pip3 which is a python package management tool.
+
+To install python 3.4 on your Raspbian Jessie, type the following:
+
+```shell
+sudo apt-get install python3 python3-dev
+wget https://bootstrap.pypa.io/get-pip.py
+sudo python3 get-pip.py
+```
+
+### OpenBLAS
+OpenBLAS is a library for fast linear algebra operations, which can significantly increase the speed of your models. It is optional, but highly recommended. To install OpenBLAS, type the following.
+
+You should not use the libopenblas-dev from the Raspbian distribution channel. Instead, build it from the source to make sure it's built for ARMV6 which is our Raspberry Pi Zero runs on. 
+
+```shell
+git clone https://github.com/xianyi/OpenBLAS.git OpenBLAS
+cd OpenBLAS
+make
+sudo make install
+```
+
+The whole build process will take about 4-5 hours. The above steps will install a working copy of OpenBLAS to /opt/OpenBLAS
+
+### Numpy
+Numpy is a numerical processing package required by OpenCV and our tutorial. Here we will install a copy of Numpy package which uses the OpenBlas we installed in the last step:
+
+```shell
+git clone https://github.com/numpy/numpy.git numpy
```

Comment:
Add this to the beginning (to avoid cloning numpy in OpenBlas):
cd ~

Reply:
Good to have it. Added. Thanks!